wandb/sdk/artifacts/utils.py

```diff
@@ -0,0 +1,5 @@
+REGISTRY_PREFIX = "wandb-registry-"
+
+
+def is_artifact_registry_project(project: str) -> bool:
+    return project.startswith(REGISTRY_PREFIX)
```

Comment:
suggestion: consider moving this into [`wandb.sdk.artifacts._validators`](https://github.com/wandb/wandb/blob/main/wandb/sdk/artifacts/_validators.py) so it's not implicitly a "public" function for users (unless that's the intent).

this allows a bit more flexibility to make breaking changes to this function in the future (not expecting to, but just in case)

Reply:
got it moving over to _validators